http/src/main/resources/reference.conf

```diff
@@ -71,7 +71,8 @@ dataproc {
 
   customDataprocImage = "projects/broad-dsp-gcr-public/global/images/leo-dataproc-image-2-0-51-debian10-2023-04-12-19-18-34"
 
-  dataprocReservedMemory = 6g
+  jupyterReservedMemory = 1.5g
```

Comment:
What is the reasoning behind this memory allocation? Leaving a comment might be nice here

Reply:
Minimum suggested for Jupyter is 1.5g. I'll add a comment!